docs/security_bots.md

```diff
@@ -0,0 +1,138 @@
+         MSMTP_HOST: ${MSMTP_HOST}
+         MSMTP_PASSWORD: ${MSMTP_PASSWORD}
+         MSMTP_PORT: ${MSMTP_PORT}
+         MSMTP_STARTTLS: ${MSMTP_STARTTLS}
+         NGXBLOCKER_CRON: ${NGXBLOCKER_CRON}
+         NGXBLOCKER_CRON_COMMAND: ${NGXBLOCKER_CRON_COMMAND}
+         NGXBLOCKER_CRON_START: ${NGXBLOCKER_CRON_START}
+         NGXBLOCKER_ENABLE: ${NGXBLOCKER_ENABLE}
+       ports:
+         - "80:80"
+         - "443:443"
+       volumes:
+         - ${ARCHIPELAGO_ROOT}/config_storage/nginxconfig/template:/etc/nginx/templates
+         - ${ARCHIPELAGO_ROOT}/drupal:/var/www/html:cached
+         - ${ARCHIPELAGO_ROOT}/data_storage/ngnixcache:/var/cache/nginx
+         - ${ARCHIPELAGO_ROOT}/data_storage/letsencrypt:/etc/letsencrypt
+         - ${ARCHIPELAGO_ROOT}/config_storage/nginxconfig/bots.d:/etc/nginx/bots.d
+   ```
+3. Bring the docker ensemble down, pull the new image, and bring the ensemble back up:
+   ```shell
+   docker compose down && docker compose pull && docker compose up -d
+   ```
+
+    !!! note
+    
+        If using an older version of docker, don't forget the hyphen:
+        ```shell
+        docker-compose down && docker-compose pull && docker-compose up -d
+        ```
+
+4. Run the install script for the bot blocker in the default dry run mode and review the output:
+   ```shell
+   docker exec -ti esmero-web bash -c "/usr/local/sbin/install-ngxblocker"
+   ```
+5. If everything looks ok, run with the execute flag:
+   ```shell
+   docker exec -ti esmero-web bash -c "/usr/local/sbin/install-ngxblocker -x"
+   ```
+6. Run the setup script for the bot blocker in the default dry run mode and review the output:
+   ```shell
+   docker exec -ti esmero-web bash -c "/usr/local/sbin/setup-ngxblocker -v /etc/nginx/templates -e .copy"
+   ```
+7. If everything looks ok, run with the execute flag:
+   ```shell
+   docker exec -ti esmero-web bash -c "/usr/local/sbin/setup-ngxblocker -v /etc/nginx/templates -e .copy -x"
+   ```
+8. Enable the bot blocker and cron (if applicable):
+   ```dotenv title=".env" hl_lines="7 10"
+   MSMTP_ACCOUNT=SMTP_ACCOUNT_NAME
+   MSMTP_EMAIL=repositorysupport@metro.org
+   MSMTP_HOST=smtp.metro.org
+   MSMTP_PASSWORD=YOUR_SMTP_PASSWORD
+   MSMTP_PORT=SMTP_PORT
+   MSMTP_STARTTLS=on
+   NGXBLOCKER_ENABLE=true
+   NGXBLOCKER_CRON=00 22 * * *
+   NGXBLOCKER_CRON_COMMAND=/usr/local/sbin/update-ngxblocker -x
+   NGXBLOCKER_CRON_START=true
+   ```
+
+    !!! note
+    
+        If `MSMTP_EMAIL` is blank and cron is enabled the flag for sending email notifications will be skipped.
+
+9. Bring the Docker ensemble down and back up again:
+   ```shell
+   docker compose down && docker compose up -d
+   ```
+
+    !!! note
+    
+        If using an older version of docker, don't forget the hyphen:
+        ```shell
+        docker-compose down && docker-compose up -d
+        ```
+
+10. There is a small bug that causes environment substitution to fail on the NGINX container so we need to run it manually:
```

Comment:
Can you expand (for me) on this? I thought we were already doing .env usage on our templates?

Reply:
Hi @DiegoPino. We are, but for reasons I'm not entirely clear on yet, the substitution doesn't trigger on up at this step.

Reply:
Could you expand/help me understand what this step entails/what is different from the built in ENV expansion shipped? I don't feel confident yet to merge this with a small bug around. Basically I have questions. Let's go step by step during our 1:1 if that helps

Reply:
Yes, sorry, "bug" is maybe not the right word. This is the built-in env expansion that doesn't seem to always trigger. I'm not a 100% sure if that's intended or if there's a small bug in the NGINX image itself, or if it has something to do with Docker, but in any case, will explain more in our call.